k2/csrc/fsa_algo.h

```diff
@@ -239,6 +239,19 @@ FsaVec LinearFsas(Ragged<int32_t> &symbols);
 Ragged<int32_t> ShortestPath(FsaVec &fsas,
                              const Array1<int32_t> &entering_arcs);
 
+/* Compute the union of all fsas in a FsaVec.
+
```

Comment:
OK... this is not the most general interface that we might need, but could be OK for now.
We might eventually need to do a union (FsaVec, FsaVec -> FsaVec), or (FsaVec, Fsa -> FsaVec);
or in fact (FsaVec, [Ragged<Int> that shows how the Fsas in the FsaVec are to be grouped]) -> FsaVec.

But we can add other version later or extend the interface.